torch/ao/quantization/quantize_fx.py

```diff
@@ -63,59 +63,14 @@ def _fuse_fx(
         graph_module, is_qat, fuse_custom_config, backend_config)  # type: ignore[operator]
 
 
-class Scope(object):
-    """ Scope object that records the module path and the module type
-    of a module. Scope is used to track the information of the module
-    that contains a Node in a Graph of GraphModule. For example::
-
-        class Sub(torch.nn.Module):
-            def forward(self, x):
-                # This will be a call_method Node in GraphModule,
-                # scope for this would be (module_path="sub", module_type=Sub)
-                return x.transpose(1, 2)
-
-        class M(torch.nn.Module):
-            def __init__(self):
-                self.sub = Sub()
-
-            def forward(self, x):
-                # This will be a call_method Node as well,
-                # scope for this would be (module_path="", None)
-                x = x.transpose(1, 2)
-                x = self.sub(x)
-                return x
-
-    """
-
+class Scope(torch.fx.proxy.Scope):
```

Comment:
I added this class definition back to quantize_fx.py for BC.  It just exposes the base class implementation to this module.

Do I also need to add it to quantization/fx/tracer.py ?

Reply:
yeah, please add to quantization/fx/tracer.py as well, also I think you can just import the class from torch.fx.proxy instead of redefining the object?

Reply:
actually the definitions in this file can probably be removed, looks like I forgot to remove them in previous refactor: https://github.com/pytorch/pytorch/commit/1a7e560adecb0192f69f4d05b990800b60dc380b, I think we can just import Scope and ScopeContextManager from fx.tracer to maintain BC and remove the definitions